File: 코딩테스트문제/설탕배달.py
n=int(input())
count=0
if n%3==0:
    count+=n//3
    n=n%3
else:
    count=n//5
    n=n%5
    count+=n//3
    n=n%3


if n!=0:
    print(-1)
else:
    print(count)

File: 자바스크립트문제/백준문제/설탕배달.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : __dirname+"/input.txt";
let input = fs.readFileSync(filePath).toString().split("\n");

function solution(){
    let kg=+input[0]
    let count=0
    while(kg>0){
        if(kg%5===0){
            kg-=5
        }
        else{
            kg-=3
        }
      count++
    }
    kg===0?console.log(count):console.log(-1)
}

solution()